### ISSUE TITLE
Figure title for images within tables doesn't increment in number

### ISSUE THREAD

**Issue by DavidGamba** [asciidoctor/asciidoctor#1314]
When setting a title for images within a table a get the following result:

![adoc-figure-in-table-issue](https://cloud.githubusercontent.com/assets/1115287/7141574/c0e396c6-e28f-11e4-8eb5-7bdffec6292b.png)

Here is the code I used for that table:

```
[cols="a,a"]
|===

|.pass:normal[{product} server 1 usage]
image::images/graph1.svg[]
|.pass:normal[{product} server 2 usage]
image::images/graph2.svg[]

|.pass:normal[{product} server 3 usage]
image::images/graph3.svg[]
|.pass:normal[{product} server 4 usage]
image::images/graph4.svg[]

|.pass:normal[{product} server 5 usage]
image::images/graph5.svg[]
|.pass:normal[{product} server 6 usage]
image::images/graph6.svg[]

|===
```

**Comment by DavidGamba** [comment]
Also, it doesn't respect when I set `:figure-caption!:` at the top of the document. I have to set it on every cell.

**Comment by mojavelinux** [comment]
AsciiDoc table cells are handled using a discrete, nested document. We'll need to find a way to pass a reference of the counters to the nested document so that they act as one.

**Comment by mojavelinux** [comment]
This is a duplicate of #1942 and was resolved in 1.5.6. The problem was that counters were not being shared with the inner documents. Now they are.
